portal-api/api-tests/v1/utilisation-report-service/utilisation-report-upload.api-test.js

```diff
@@ -8,6 +8,21 @@ const { DB_COLLECTIONS } = require('../../fixtures/constants');
 
 const { as, postMultipartForm } = require('../../api')(app);
 const { PAYMENT_REPORT_OFFICER } = require('../../../src/v1/roles/roles');
+const { uploadFile } = require('../../../src/drivers/fileshare');
+
+const mockFileUpload = {
+  folder: 'folder_name',
+  filename: 'test_file.csv',
+  fullPath: 'folder_name/test_file.csv',
+  url: 'https://azure/utilisation-reports/folder_name/test_file.csv',
+};
```

Comment:
We create a mock value for this `AzureFileInfo` type in quite a few places. Are we able to create a `MOCK_AZURE_FILE_INFO` (or similar) in `portal-api/test-helpers`?

This would also make things much easier to update if the type ever changed as we'd only need to update one place to get all out tests using the updated type. We should also add this to `MOCK_UTILISATION_REPORT` as that should now have the extra `azureFileStorage` too.